common/mount.py

```diff
@@ -437,13 +431,9 @@ def __init__(self,
         self.mountproc = None
         self.symlink_subfolder = None
 
-        self.config = cfg
-        if self.config is None:
-            self.config = config.Config()
+        self.config = cfg or config.Config()
 
-        self.profile_id = profile_id
-        if self.profile_id is None:
-            self.profile_id = self.config.currentProfile()
+        self.profile_id = profile_id if profile_id is not None else self.config.currentProfile()
```

Comment:
Missed one? Or am I wrong?

```suggestion
        self.profile_id = profile_id or self.config.currentProfile()
```

Reply:
yeah, you are right
updated

Reply:
I don't see this line updated. Am I wrong?